helm/dagster/schema/schema_tests/test_user_deployments.py

```diff
@@ -933,6 +933,61 @@ def test_annotations(template: HelmTemplate, include_config_in_launched_runs: bo
     else:
         _assert_no_container_context(user_deployments[0])
 
+@pytest.mark.parametrize("include_config_in_launched_runs", [False, True])
+def test_tolerations(template: HelmTemplate, include_config_in_launched_runs: bool):
+    name = "foo"
+
+    tolerations = kubernetes.Tolerations.parse_obj([{"my-toleration-key": "my-toleration-val", 
+                                                        "toleration-operator": "Equal", 
+                                                        "toleration-value": "1", 
+                                                        "toleration-effect": "NoSchedule"}])
+
```

Comment:
is it possible to add a test case that includes both tolerations and annotations to ensure that we handle the case above?

Reply:
I guess the above is not as relevant if it's moving off of pod_template_spec_metadata.